tests/lib/json/src/main.c

```diff
@@ -291,6 +329,77 @@ static void test_json_obj_arr_encoding(void)
 		     "Encoded array of objects is not consistent");
 }
 
+static void test_json_arr_obj_decoding(void)
+{
+	int ret;
+	struct obj_array obj_array_array_ts;
+	char encoded[] ="[{\"height\":168,\"name\":\"Simón Bolívar\"},"
```

Comment:
Looks like I missed a space here after the =, sorry about that! Should be `char encoded[] = "[...` instead of `char encoded[] ="[...`